### PR TITLE
[DEVSU-2342] patients view reactivate

### PR DESCRIPTION
**Pull request by kttkjl**

### CODE REVIEW

File: app/views/PatientsView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+              analysisBiopsy: report.biopsyName,
+              reportType: report.type === 'genomic' ? 'Genomic' : 'Targeted Gene',
+              state: report.state,
+              caseType: report.patientInformation.caseType,
+              project: report.projects.map((project) => project.name).sort().join(', '),
+              physician: report.patientInformation.physician,
+              analyst: analyst ? `${analyst.firstName} ${analyst.lastName}` : null,
+              reportIdent: report.ident,
+              tumourType: report.patientInformation.diagnosis,
+              date: report.createdAt,
```
Comment (bnguyen-bcgsc) [inline app/views/PatientsView/index.tsx:51]: I think there are 2 missing columns here: Bioinformatician and Reviewer

File: app/views/PatientsView/index.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  };
+
+  const defaultColDef = {
+    sortable: true,
+    resizable: true,
+    filter: true,
+  };
+
+  return (
+    <div className="ag-theme-material reports-table__container">
```
Comment (bnguyen-bcgsc) [inline app/views/PatientsView/index.tsx:96]: Should we use the ReportsTable component here now that it's available?

**Review by bnguyen-bcgsc — CHANGES_REQUESTED** [review]

File: app/views/PatientsView/index.tsx (hunk 2 above)
Comment (kttkjl) [reply app/views/PatientsView/index.tsx:96]: Oh yeah, definitely

File: app/views/PatientsView/index.tsx (hunk 1 above)
Comment (kttkjl) [reply app/views/PatientsView/index.tsx:51]: good catch, just illustrates how outdated this component was lol

Comment (kttkjl) [reply app/views/PatientsView/index.tsx:54]: [4548417](https://github.com/bcgsc/pori_ipr_client/pull/495/commits/4548417c7d667f901d6c78c8553a1810ea04af1a)

**Review by bnguyen-bcgsc — APPROVED** [review]

**Pull request merged**